update glaossary 121 to new (#72)
global-121/manual: docs/glossary-121.md

| :------------------    | :------------------------------------------------------------------- |
| **121 Platform**       | (“one to one”) is a platform that helps project managers by increasing efficiency in setting up and executing a cash program Information Management (IM) system.  |
| **CVA**                | Cash and Voucher Assistance, referring to Cash Transfer Programming (CTP). |
| **Project**            | A program created by users in the platform. |
| **FSP**                | Financial Service Provider - an entity that provides financial services, which may include digital payment services.|
| **Budget**             | Amount of money in the project that will be disbursed to included people/households.|
| **Registrations**      | People or households that registered, subscribed to a project.|
| **Import**             | Adding or uploading data from excel file or interview forms e.g. "import registrations".|
| **Export**             | Downloading content from the 121 Platform.|
| **Actions**            | Where a user can do something that affects one or multiple registrations, e.g. sending a message, adding a note, or updating the information.|

> | **Filter by**          | A place for users to extract registrations to find a group of registrations or a single registration that they are looking to perform an action on e.g. filter registrations that received 2 payment and include in a new payment.|

Reviewer: ```suggestion
| **Filter by**          | Users can use this to find a group of registrations or a single registration that they are looking to perform an action on e.g. filter registrations that received 2 payment and include in a new payment.|
```

> | **Quick search**       | A filter search that shows results of multiple columns instead of filtering by 1 column. This is a fast way to find a single or small group of registrations.|

Reviewer: ```suggestion
| **Quick search**       | A filter search that shows results of multiple columns instead of filtering in just one column. This is a fast way to find a single or small group of registrations.|
```

| **Type / Select value**| when a column is selected to filter by, users can then type or select the word/ numerical value they wish to filter the table by.|
| Payment or Payments | A collection of transfers to one or multiple registrations. |

> | **Payment or payments** | A collection of transfers to one or multiple registrations. |

Reviewer: lets be consistent in registrations vs person/household
